src/utils/isClass.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+export default function isClass(target: any) {
+  return typeof target === 'function' && /^\s*class\s+/.test(target.toString())
```

Comment:
I don't think this will work for transpiled classes?

Reply:
Right, forgot about that and according to https://github.com/babel/babel/issues/5640 it's not possible right now.